sdk/trace/simple_span_processor.go

```diff
@@ -33,11 +38,26 @@ func NewSimpleSpanProcessor(exporter SpanExporter) SpanProcessor {
 	ssp := &simpleSpanProcessor{
 		exporter: exporter,
 	}
+
+	var err error
+	ssp.inst, err = observ.NewSSP(nextSimpleProcessorID())
+	if err != nil {
+		otel.Handle(err)
```

Comment:
I think we can directly return the error here.

Reply:
No, please ignore this question. If adjusted in this way, it will not be backward compatible.